packages/transport-native/tsconfig.build.json

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
nitpick: unnecessary line, eslint didnt complain?

Reply:
it did not I think it is not checking .json... but fixed now